modules/infra/outputs.tf

```
@@ -80,3 +80,8 @@ output "monitoring_bucket" {
   description = "Monitoring Bucket"
   value       = module.storage.info.s3.buckets.monitoring.bucket_name
 }
+
+output "cost_reporting" {
```

keeping the nomenclature consistent:
```suggestion
output "cost_usage_report" {
```